src/components/Character/styles.scss

```diff
@@ -1,19 +1,189 @@
+	}
+
+		.content {
+			position: relative;
+			margin-top: 40vh;
+		}
+	
+			.name {
+				margin-bottom: 0.1em; // GUESS
+				font-size: 5em;
+				font-family: var(--font_family__heading);
+			}
+
+			.nickname {
+				margin-bottom: 0.5em; // GUESS
+				font-size: 2.6em;
+				font-family: var(--font_family__heading);
+			}
+
+			.actor {
+				margin-bottom: 1em; // GUESS
+				font-size: 1em;
+				font-family: var(--font_family__body);
+				font-weight: normal;
+				letter-spacing: 0.25em;
+				text-transform: uppercase;
+			}
+
+			.body {
+				margin-bottom: 4em; // GUESS
+				line-height: 1.5;
+				font-family: var(--font_family__body);
+			}
+
+		.content::after {
+			content: "";
+			display: block;
+			position: fixed;
+			width: 100%;
+			height: 12em;
+			bottom: 0;
+			left: 0;
+			background-image: linear-gradient(
+			    rgba(237, 20, 61, 0) 0%,
+			    rgba(237, 20, 61, 1) 50%,
+			    rgba(237, 20, 61, 1) 100%	
+			);
+		}
+
+		.controls {
+			display: flex;
+			position: sticky;
+			width: 100%;
+			max-width: 25rem; // GUESS
+			bottom: 10vw; // GUESS!
+			justify-content: space-between;
+			font-size: 1.1em;
+			font-family: var(--font_family__heading);
 		}
-		&.previous {
-			font-size: 1em;
+
+			.button {
+				display: flex;
+				align-items: center;
+				text-decoration: none;
+				--button_margin: 1em;
+
+				&::before, 
+				&::after {
+					display: inline-block;
+					font-size: 1.36em;
+					width: 1em;
+					height: 1.5em;
+					background-color: lime;
+				}
+
+				&.previous {
+					&::before {
+						content: "";
+						margin-right: var(--button_margin);
+					}
+				}
+
+				&.next {
+					&::after {
+						content: "";
+						margin-left: var(--button_margin);
+					}
+				}
+			}
+
+// Mobile landscape
+@media (orientation: landscape) {
+	.wrapper {
+		max-height: 100%;
+		padding: 5%; //GUESS
+	}
+
+		.container {
+			width: 50%; 
+			height: 100%;
+			--button__bottom_spacing: 5vw;
 		}
+
+			.content {
+				margin-top: 0;
+			}
+
+				.body {
+					height: 25vh; //GUESS
+					margin-bottom: 2em; //GUESS
+					overflow-y: scroll;
+				}
+
+			.controls {
+				bottom: var(--button__bottom_spacing); //GUESS
+			}
+
+			.content::after {
+				position: fixed;
+				bottom: var(--button__bottom_spacing); //GUESS
+				height: 6em; //GUESS
+				pointer-events: none;
+				background-image: linear-gradient(
+				    rgba(237, 20, 61, 0) 0%,
+				    rgba(237, 20, 61, 1) 50%,
+				    rgba(237, 20, 61, 1) 100%	
+				);
+			}
+}
+
+// Tablet portrait
+@media (min-width: 768px) {
+	.wrapper {
+		padding: 10%;
 	}
+}
+
+// Tablet landscape
+@media (min-width: 768px) and (orientation: landscape) {
+	.wrapper {
+		justify-content: center;
+	}
+
+		.container {
+			--button__bottom_spacing: 15vw; //GUESS
+		}
+}
 
+// Desktop portrait
+@media (min-width: 1200px) and (max-aspect-ratio: 8/9) {
+	
+}
 
+// Desktop landscape
+@media (min-width: 1200px) {
+
+	.name {
+		font-size: 8rem;
+	}
 
+	.nickname {
+		font-size: 3.25rem;
```

Comment:
```suggestion
		font-size: 3.25em;
```
!! 🚨 unnecessary `rem` 🚨 !!
This is definitely `em` as it's this size because it's proportion of the body - if you changed the `font-size` of the `.wrapper` then you'd definitely want this one to scale proportionately instead of staying the same size